tekton/src/params/install.yml.j2

```diff
@@ -236,20 +236,38 @@
 # -----------------------------------------------------------------------------
 - name: uds_contact_email
   type: string
+  default: ""
```

Comment:
stick with just the uds params ... which we can map to the DRO env vars in the task to avoid needing to duplicate all these inputs

Reply:
done